build/gen_stub.php

```diff
@@ -131,6 +157,8 @@ class Context {
     public $forceRegeneration = false;
     /** @var iterable<ConstInfo> */
     public iterable $allConstInfos = [];
+    /** @var FileInfo[] */
+    public array $parsedFiles = [];
```

Comment:
The property type above (`$allConstInfos`) is an issue in stubs which is about to be fixed in a soon to be created PR of mine. The issue with it is that the gen_stub.php is part of the build system which is compatible with PHP 7.1 as of now.